src/app/icd/ICDEventManager.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+#include <app/icd/ICDEventManager.h>
+#include <app/icd/ICDManager.h>
+#include <lib/core/CHIPError.h>
+#include <platform/CHIPDeviceEvent.h>
+
+namespace chip {
+namespace app {
+
+/**
+ * @brief ICDEventManager class is responsible of processing Platform Events that affect an ICDs behaviour
```

Comment:
```suggestion
 * @brief ICDEventManager class is responsible of processing Platform Events that affect an ICD's behaviour
```
I commented on this last time, too....

Reply:
didn't see the comment... my bad.